types/stripe/index.d.ts

```diff
@@ -5,11 +5,12 @@
 //                 Amrit Kahlon <https://github.com/amritk/>
 //                 Adam Cmiel <https://github.com/adamcmiel>
 //                 Justin Leider <https://github.com/jleider>
+//                 Kamil Gałuszka <https://github.com/galuszkak>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
```

Comment:
So you're saying that Stripe has possibly three different API surfaces based on whether you've loaded v2, v3, or both? I don't quite understand their loading model